src/components/Search.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Search = ({ onSearch, placeholder = 'Søg brugere' }) => {
+  const searchHandler = (event) => {
+    // parse the string if needed here
```

Comment:
The component should always pass the final result instead of event; also it might need to do data validation or parsing here